### PR TITLE
Zendesk touch

### PR DESCRIPTION
**Pull request by arthurgetaround**
@fireplacetv

### CODE REVIEW

File: agent_touch.view.lkml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+view: agent_touch {
+  sql_table_name: dbt_arthur_common.zendesk_ticket_touch ;;
```
Comment (fireplacetv) [inline agent_touch.view.lkml]: Let's point this to the prod table

File: agent_touch.view.lkml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+
+  dimension: id {
+    description: "ID for the touch of a ticket"
+    primary_key: yes
+    type: number
+    sql: ${TABLE}."id" ;;
+  }
```
Comment (fireplacetv) [inline agent_touch.view.lkml:11]: Will anyone ever need this on its own? I think we can probably leave this hidden.

File: agent_touch.view.lkml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+  dimension: agent_name {
+    description: "Full name of Agent who touched the ticket"
+    group_label: "Agent"
+    type: string
+    sql: ${TABLE}."agent_name" ;;
+  }
+
+  dimension: ticket_id {
```
Comment (fireplacetv) [inline agent_touch.view.lkml]: We only need this for the join, so we can leave it hidden. Also, might as well move it up to the top with the other ID.

File: agent_touch.view.lkml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      day_of_month,
+      year,
+      week_of_year,
+      month_num,
+      quarter_of_year
+    ]
+    sql: ${TABLE}.created_at ;;
+  }
+
+  dimension: agent_email {
```
Comment (fireplacetv) [inline agent_touch.view.lkml:70]: Since there are only 4 fields, I actually wouldn't bother grouping these since it just adds extra clicks for the user.

**Review by fireplacetv — CHANGES_REQUESTED** [review]
Looks good--just requesting a couple small changes.

**Comment by arthurgetaround** [comment]
@fireplacetv Addressed!

**Review by fireplacetv — APPROVED** [review]
lgtm!

**Review by jaywmehta — APPROVED** [review]
LGTM

**Pull request merged**